tests/DoctrineMongoODMModuleTest/Doctrine/CliTest.php

```diff
@@ -26,7 +34,7 @@
  * @link    http://www.doctrine-project.org/
  * @author  Adam Homsi <adam.homsi@gmail.com>
  */
-class CliTest extends PHPUnit_Framework_TestCase
+final class CliTest extends \PHPUnit_Framework_TestCase
```

Comment:
@gianarb  That is completely false. This entire namespace is only present in `autoload-dev`, ergo it is not even accessible to users. They cannot be using this.